src/main/java/org/opensearch/ad/task/ADBatchTaskRunner.java

```diff
@@ -1087,26 +1085,12 @@ private void detectAnomaly(
                 anomalyResults.add(anomalyResult);
             } else {
                 double[] point = feature.getProcessedFeatures().get();
-                double score = rcf.getAnomalyScore(point);
-                rcf.update(point);
-                double grade = 0d;
-                double confidence = 0d;
-                if (!adTaskCacheManager.isThresholdModelTrained(taskId)) {
-                    if (adTaskCacheManager.getThresholdModelTrainingDataSize(taskId) < THRESHOLD_MODEL_TRAINING_SIZE) {
-                        if (score > 0) {
-                            adTaskCacheManager.addThresholdModelTrainingData(taskId, score);
-                        }
-                    } else {
-                        logger.debug("training threshold model");
-                        threshold.train(adTaskCacheManager.getThresholdModelTrainingData(taskId));
-                        adTaskCacheManager.setThresholdModelTrained(taskId, true);
-                    }
-                } else {
-                    grade = threshold.grade(score);
-                    confidence = threshold.confidence();
-                    if (score > 0) {
-                        threshold.update(score);
-                    }
+                AnomalyDescriptor descriptor = trcf.process(point, 0);
```

Comment:
How about adding some comments to explain what 0 means?

Reply:
yes, added

// 0 is placeholder for timestamp. In the future, we will add
// data time stamp there.

Reply:
The timestamp means model's last update time?

Reply:
it can be the data start or end time in anomaly result index.  We can use either, just need to be consistent.

Reply:
+1